src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -1564,6 +1564,19 @@ void set_camera_dof_params(
     }
 }
 
+void set_camera_film_params(
```

Comment:
Nice! I would suggest changing `set_camera_dof_params()` to no longer set film params, and calling this function instead.